src/SAML2/SignedElementHelper.php

```diff
@@ -9,60 +9,44 @@
 use RobRichards\XMLSecLibs\XMLSecurityKey;
 
 /**
- * Helper class for processing signed elements.
- *
- * Can either be inherited from, or can be used by proxy.
+ * Helper trait for processing signed elements.
  *
  * @package SimpleSAMLphp
  */
-class SignedElementHelper extends SignedElement
+trait SignedElementHelper
```

Comment:
If we want to follow PSR naming conventions, we should suffix this class with 'Trait' and rename the file accordingly

Reply:
True! I was thinking about that, but was uncertain whether the pattern was followed for traits as well...

Reply:
It is, and I have fixed this in the simplification-PR, so basically this is _Resolved_ now..
References to traits need to be fixed in this PR though..